sympy/stats/tests/test_continuous_rv.py

```diff
@@ -429,7 +429,8 @@ def test_gumbel():
     mu = Symbol("mu")
     x = Symbol("x")
     X = Gumbel("x", beta, mu)
-    assert simplify(density(X)(x)) == exp((beta*exp((mu - x)/beta) + mu - x)/beta)/beta
+    assert simplify(density(X)(x)) == exp(-exp(-(-mu + x)/beta) - (-mu + x)/beta)/beta
```

Comment:
```suggestion
    assert simplify(density(X)(x)) == \
        simplify(exp(-exp(-(-mu + x)/beta) - (-mu + x)/beta)/beta)
```
I guess there is a problem with canonicalization, here
This may make the test pass, at least

Reply:
Or @sidhantnagpal 's suggestinn of using string comparison may be better.